app/services/events_service.rb

```diff
@@ -2,7 +2,18 @@
 
 class EventsService < BaseService
   def validate_params(params:)
-    mandatory_augments = %i[transaction_id customer_id code]
+    mandatory_augments = %i[transaction_id code]
```

Comment:
Is that a typo to `mandatory_arguments`? 😅

Reply:
Probably mine :facepalm: :D